Resources/Textures/Structures/Machines/diagnoser.rsi/meta.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": 1,
+  "license": "CC-BY",
+  "copyright": "Created by Willhudson#4576 (Discord user id: 935437363180613672) in the SS14 Discord",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
+    },
+    {
+      "name": "unlit"
+    },
+    {
+      "name": "running",
+      "delays": [
+        [
+          0.1785,
+          0.1785,
+          0.1785,
+          0.1785,
+          0.1785,
```

Comment:
what

Reply:
5 seconds (length of the delay) / number of frames in the animation